client/src/components/list-container/list-controls-component/ListControlsComponent.jsx

```diff
@@ -0,0 +1,47 @@
+import { useCallback, useState } from "react";
+import TextField from "monday-ui-react-core/dist/TextField";
+import Button from "monday-ui-react-core/dist/Button";
+import Add from "monday-ui-react-core/dist/icons/Add";
+import PropTypes from "prop-types";
+import ListApiService from "../../../services/list-api-service";
+import { addAnotherItem } from "../../../actions/add-todo-action";
+import styles from "./ListControlsComponent.module.scss";
+
+function ListControlsComponent({ addAnotherItem, addItemNew }) {
+	const [name, setName] = useState("");
+
+	const onInputChange = useCallback((value) => {
+		setName(value);
+	}, []);
+
+	const addItem = useCallback(async () => {
+		setName("");
+		addAnotherItem(name);
+		// onChange(addItemNew);
```

Comment:
make sure not to leave comments on code, if its for "saving the code" for some reason then you'll be able to see it in the git history in the future